cypress/e2e/shared/dashboardsVariables.test.ts

```diff
@@ -176,13 +176,13 @@ describe('Dashboard - variable interactions', () => {
 
           // Make sure typeAhead input box is rendered and is visible
           cy.getByTestID(
-            `variable-dropdown-input-typeAhead--${bucketVarName}`
+            `variable-dropdown--${bucketVarName}--typeAhead-input`
```

Comment:
These are just testID changes, seem like a lot but basically it's just changing the testID order.